firefox_puppeteer/tests/test_windows.py

```diff
@@ -13,6 +13,10 @@
 
 class TestWindows(FirefoxTestCase):
 
+    def tearDown(self):
+        self.windows.close_all([self.browser])
+        FirefoxTestCase.tearDown(self)
```

Comment:
Better do a try/finally here, so we can be sure that we always call tearDown of the super class.